docs/reference/callbacks.txt

```diff
@@ -13,7 +13,7 @@ Callbacks
 Document Callbacks
 ==================
 
-Mongoid supports the following callbacks for :doc:`documents </tutorials/documents>`:
+Mongoid supports the following callbacks for :doc:`documents </tutorials/mongoid-documents>`:
```

Comment:
I don't think this is right: https://github.com/mongodb/mongoid/blob/master/docs/tutorials/documents.txt

Reply:
Try for yourself:
* https://docs.mongodb.com/mongoid/current/tutorials/documents/
* https://docs.mongodb.com/mongoid/current/tutorials/mongoid-documents/

Reply:
Oops, you are correct. The docs which are online at docs.mongodb.com are stale. Fixed this so resolving.

Reply:
Yes, the master documentation on `docs.` isn't regularly updated (in fact in the last year+ it's only really been updated when we release, thus in practice it is the same as `current` most of the time).